CONTRIBUTING.md

```diff
@@ -28,7 +28,10 @@ git clone https://github.com/scalameta/metadoc.git
 cd metadoc
 git submodule init
 git submodule update
+
 npm install -g yarn
+yarn config set ignore-engines true # To avoid error: engine node is incompatible with this version
```

Comment:
Is this for the `upath` dependency? I ended up using `nvm` to install node version 9.

Reply:
Yes, it was upath. I have node v10 installed, ignoring the warning fixed the problem so I'm happy 😊